src/zac/demo/mor/tests/factories.py

```diff
@@ -0,0 +1,25 @@
+import json
+
+from factory.django import DjangoModelFactory as Dmf
+from zac.accounts.models import User
+from ...models import SiteConfiguration
+
+
+class UserFactory(Dmf):
+
+    class Meta:
+        model = User
+
+    username = 'test'
```

Comment:
don't use constants, but factory-boy structures. `factory.Faker('username')` would be okay here.